manifests/overlays/dev/kfdef.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: kfdef.apps.kubeflow.org/v1
+kind: KfDef
```

Comment:
This fails unless ODH was installed previously:

```
error: unable to recognize "STDIN": no matches for kind "KfDef" in version "kfdef.apps.kubeflow.org/v1"
```

I'm not sure how to best fix that without complicating things too much though. Thoughts?

Reply:
Yeah.. you need to wait for the operator to install and create the CRDs.. re-issuing the command after few minutes should pass. In argocd we can solve this by using sync waves, here we may want to split the manifests into 2 overlays and apply them separately, wdyt?

Reply:
What about this @codificat ? I've split the overlay into 2: `dev/phase_00` which subscribes the cluster to operators and `dev/phase_01` which deploys kfdef, pipeline and task resources. The `make deploy` command executes both phases and waits in between until CRDs exist.

Reply:
Thanks, that should work better.

It still failed to complete because ODH would not deploy - I opened https://github.com/operate-first/apps/pull/1605 about that, but that's another story and I believe we are fine to get this in